Dockerfile

```diff
@@ -0,0 +1,43 @@
+# Copyright 2020 SUSE
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+ARG FINAL_IMAGE_BASE=registry.suse.com/suse/sle15:15.2.8.2.763@sha256:ec381220e66096967e4babb4c3703fd68e4466943984b63770ac3b82e75c1830
+
+FROM opensuse/tumbleweed AS staging
```

Comment:
Alternatively, we can use the stable release + [obs://devel:languages:go](https://build.opensuse.org/project/show/devel:languages:go); however, I'm not sure that's any _better_ (since using Tumbleweed should mean that's at least been used to package other things, so the toolchain actually works, which using the devel project doesn't guaranteee).

In conclusion, my idea is bad and what you have is better.